.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+# Description
```

Comment:
```suggestion
# References
<!-- Any issues or pull requests relevant to this pull request -->

# Description
```
We also have `References` setup in other repos (i.e. python ffi)

Reply:
Yeah, I removed it since most contributors are confused about how to use it.

Reply:
uh ok, got it